src/desktop/config.coffee

```diff
@@ -68,8 +68,10 @@ module.exports =
   GEOIP_ENDPOINT: 'https://artsy-geoip.herokuapp.com/'
   GOOGLE_ADWORDS_ID: null
   GOOGLE_MAPS_API_KEY: null
-  INTERCOM_APP_ID: null
-  INTERCOM_ENABLED: false
+  INTERCOM_GP_APP_ID: null
+  INTERCOM_GP_ENABLED: false
+  INTERCOM_BN_APP_ID: null
+  INTERCOM_BN_ENABLED: false
```

Comment:
Maybe we can name them less specific, such as `INTERCOM_PARTNER_...` and `INTERCOM_USER_...` or `INTERCOM_SELLER_...` and `INTERCOM_BUYER_...`.

Reply:
+1
or `..._PARTNER_...` and `..._COLLECTOR_..`

Reply:
Yea, I like `seller/buyer` over `partner/collector`. make everything generic!